server/src/main/java/org/elasticsearch/common/unit/ByteSizeValue.java

```diff
@@ -264,7 +264,7 @@ private static ByteSizeValue parse(final String initialInput, final String norma
                          initialInput, settingName);
                     return new ByteSizeValue((long) (doubleValue * unit.toBytes(1)));
                 } catch (final NumberFormatException ignored) {
-                    throw new ElasticsearchParseException("failed to parse [{}]", e, initialInput);
+                    throw new ElasticsearchParseException("failed to parse setting [{}] with value [{}]", e, settingName, initialInput);
```

Comment:
I left this in this PR, but can certainly split it out for greater visibility if you prefer.

Reply:
Yeah, let's take this to another PR so that the core/infra team can have visibility on the change.